examples/custom_shaders_uniforms/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ include_directories(SYSTEM
   ${PROJECT_BINARY_DIR}
 )
 
-find_package(ignition-rendering4)
+find_package(ignition-rendering6)
```

Comment:
This change is not related to this PR I prefer to remove it from this PR

Reply:
Removed the commit - https://github.com/ignitionrobotics/ign-rendering/compare/617af60fdaa475ec13612f41f1068d574df12b34..242888265c0c2f5cd4ac9a5612e0015fcfe65609

Reply:
created PR for this change: https://github.com/ignitionrobotics/ign-rendering/pull/343